### PR TITLE
Fix AWS Observability Quickstart link

### PR DESCRIPTION
**Pull request by aneeshep**
The current link to AWS Observability Quickstart is broken. This commit will fix the broken link

## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [ ] Yes, I've run `yarn install`
- [x] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Review by kimsauce — APPROVED** [review]
good catch!

**Review by kimsauce — COMMENTED** [review]
Hi @aneeshep , the K8s quickstart link works for me (video attached below). Could you please double check on your end?

https://github.com/SumoLogic/sumologic-documentation/assets/56411016/46b4aa66-0ee7-46da-8330-dc69f22af361

**Comment by jpipkin1** [comment]
@kimsauce, we just need to remove the line for the AWS Observability Quickstart. @aneeshep marked the Kubernetes link by mistake. 

@JV0812 removed the AWS Observability Quickstart earlier, but forgot to clean up some things. See the [post in the #dochub channel](https://sumologic.slack.com/archives/C0S86TM6K/p1702601731401329?thread_ts=1702595763.835559&cid=C0S86TM6K).

File: docs/get-started/quickstart.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ Contact us at the [Sumo Dojo Slack](https://sumodojo.slack.com/) or [submit a su
 
 * [Choosing a Sumo Logic Collector and Source](/docs/send-data/choose-collector-source/)
 * [AWS Observability Quickstart](/docs/observability/aws/quickstart). This guide will walk you through setting up the Sumo Logic AWS Observability integration.
```
Comment (kimsauce) [inline docs/get-started/quickstart.md:127]:
```suggestion
```

File: docs/get-started/quickstart.md [hunk 2]
```diff
@@ -124,8 +124,7 @@ Contact us at the [Sumo Dojo Slack](https://sumodojo.slack.com/) or [submit a su
 :::
 
 * [Choosing a Sumo Logic Collector and Source](/docs/send-data/choose-collector-source/)
-* [AWS Observability Quickstart](/docs/observability/aws/quickstart). This guide will walk you through setting up the Sumo Logic AWS Observability integration.
-* [Kubernetes Quickstart](/docs/observability/kubernetes/quickstart). Get up and running with the Sumo Logic Kubernetes solution in minutes.
+* [Kubernetes Quickstart](/docs/observability/kubernetes). Get up and running with the Sumo Logic Kubernetes solution in minutes.
```
Comment (kimsauce) [inline docs/get-started/quickstart.md:127]:
```suggestion
* [Kubernetes Quickstart](/docs/observability/kubernetes/quickstart). Get up and running with the Sumo Logic Kubernetes solution in minutes.
```

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
> @kimsauce, we just need to remove the line for the AWS Observability Quickstart. @aneeshep marked the Kubernetes link by mistake.
> 
> @JV0812 removed the AWS Observability Quickstart earlier, but forgot to clean up some things. See the [post in the #dochub channel](https://sumologic.slack.com/archives/C0S86TM6K/p1702601731401329?thread_ts=1702595763.835559&cid=C0S86TM6K).

gotcha - thank you so much, john!

**Pull request merged**